cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/pom.xml

```diff
@@ -1,4 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  DO NOT ALTER OR REMOTE COPYRIGHT NOTICES OR THIS HEADER.
```

Comment:
REMOTE COPYRIGHTs are everywhere :-)

Reply:
The one line I didn't copy and paste into the copyright template... 😞 - Will get this fixed